core/server/master/src/main/java/alluxio/master/file/FileSystemMasterClientServiceHandler.java

```diff
@@ -159,8 +159,10 @@ public void createFile(CreateFilePRequest request,
       StreamObserver<CreateFilePResponse> responseObserver) {
     RpcUtils.call(LOG, () -> {
       AlluxioURI pathUri = getAlluxioURI(request.getPath());
-      return CreateFilePResponse.newBuilder().setFileInfo(GrpcUtils.toProto(mFileSystemMaster
-          .createFile(pathUri, CreateFileContext.create(request.getOptions().toBuilder()))))
+      return CreateFilePResponse.newBuilder()
+          .setFileInfo(GrpcUtils.toProto(mFileSystemMaster.createFile(pathUri,
+              (CreateFileContext) CreateFileContext.create(request.getOptions().toBuilder())
```

Comment:
See other comment.

Change to the `CreatePathContext` class def removed the need for the extra cast